hs_labels/tests/test_resource_labelling.py

```diff
@@ -105,94 +120,94 @@ def test_labels(self):
         cat = self.cat  # user
         scratching = self.scratching  # resource
         cat.ulabels.label_resource(scratching, "penalty clause")
```

Comment:
Should include test to show that duplicate labels can't be assigned to a resource (duplicate record problem).